reference-data/omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -6,5 +6,5 @@
   		    http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
   		    http://www.springframework.org/schema/context
   		    http://www.springframework.org/schema/context/spring-context-3.0.xsd">
-    <context:component-scan base-package="org.bahmni.module.referencedata.web"/>
+    <context:component-scan base-package="org.bahmni.module.referencedata"/>
```

Comment:
is it not better to be more specific? if you are only scanning web layer then "...referencedata.web" would be better